Common/cpp/Fabric/ReanimatedUIManagerBinding.cpp

```diff
@@ -37,6 +38,10 @@ void ReanimatedUIManagerBinding::createAndInstallIfNeeded(
     return std::move(uiManagerBindingPublic->eventHandler_);
   }();
 
+  // We can not cache text measurements, because we need to calculate it on each animation frame.
+  CoreFeatures::cacheNSTextStorage = false;
+  CoreFeatures::cacheLastTextMeasurement = false
```

Comment:
Disabling these flags turns off caching of all text components across the whole application, even though only a small fraction of those would be animated. I'm afraid we will need to think about another solution.

Reply:
I agree with Tomek, we can't disable the whole feature for all existing text components in an application.